docs/howto/request-public-listing.md

```diff
@@ -0,0 +1,78 @@
+(make-your-charm-discoverable)=
+# Make your charm discoverable
+
+> See first: [Charm maturity](#charm-maturity)
+
+Once your charm is ready for wide production use, get it publicly listed on [Charmhub](https://charmhub.io), so that it is visible in searches done by Juju users and other charm developers.
+
+Anyone can upload a charm to Charmhub, which allows deploying the charm and viewing its information on Charmhub if you know the name of the charm. To have a charm publicly listed, meaning it will be found in searches on Charmhub and general web searches leading to Charmhub, it must pass through a lightweight review process. The goals of the review are:
+
+1. Be transparent about the capabilities and qualities of a charm.
+2. Ensure a common level of quality.
```

Comment:
I don't like this "The goals of the review are..." bit. It's awkwardly phrased and doesn't add much: the "lightweight" bit before already suggests it's not going to be anything too deep, and the note that follows emphasizes that even further. I'd cut it out or squeeze it in the note somehow.